packages/contracts/contracts/test/L2OutputOracle.t.sol

```diff
@@ -0,0 +1,119 @@
+//SPDX-License-Identifier: MIT
+pragma solidity 0.8.10;
+
+import { DSTest } from "../../lib/ds-test/src/test.sol";
+import { L2OutputOracle } from "../L1/L2OutputOracle.sol";
+
+interface CheatCodes {
```

Comment:
This interface can be imported from here: https://github.com/brockelmore/forge-std/blob/master/src/Vm.sol